src/Tasks/Microsoft.WinFx.targets

```diff
@@ -11,4 +11,33 @@
 
    <Import Project="$(MSBuildFrameworkToolsPath)\Microsoft.WinFx.targets" Condition="Exists('$(MSBuildFrameworkToolsPath)\Microsoft.WinFx.targets')" />
 
+   <!-- This target is to fix a problem with WPF TemporaryAssembly creation during markup compilation. Shipping fix with MsBuild, until 
+        we can get the right fix into .NET Framework and Windows.
+   -->
+
+   <Target Name="GenerateTemporaryTargetAssembly"
+               Condition="'$(_RequireMCPass2ForMainAssembly)' == 'true'" >
+
+      <Message Text="MSBuildProjectFile is $(MSBuildProjectFile)" Condition="'$(MSBuildTargetsVerbose)' == 'true'" />
+
+       <GenerateTemporaryTargetAssembly2
```

Comment:
Why do we consume this Task in MSBuild but we don't define it in MSBuild? Isn't assuming that the nuget .targets defining this task a bit hazardous? 
In particular, I think this sort of pattern bit us recently in the VS repo (Razzle) where we have the msbuild shipping .targets but not the nuget targets. It ended up breaking language services within VS.

Reply:
Good catch. Will dig into this.